### PR TITLE
(#5) add package installs removed from puppet-mcollective

### PR DESCRIPTION
**Pull request by ripienaar**
Signed-off-by: R.I.Pienaar <rip@devco.net>

### CODE REVIEW

File: data/Archlinux.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
 mcollective_choria::package_dependencies:
   "ruby-nats-pure": "present"
+  "mcollective": "present"
```
Comment (ripienaar) [inline data/Archlinux.yaml]: @bastelfreak is this still right? a package called mcollective to install there? This is moving from here https://github.com/choria-io/puppet-mcollective/pull/283

File: data/FreeBSD.yaml [hunk 2]
```diff
@@ -1,3 +1,5 @@
 ---
+mcollective_choria::package_dependencies:
+  "mcollective": "present"
```
Comment (ripienaar) [inline data/FreeBSD.yaml]: @smortex is this still right for FreeBSD and OpenBSD? to install a package called `mcollective`? This is removed from https://github.com/choria-io/puppet-mcollective/pull/283

File: data/Archlinux.yaml (hunk 1 above)
Comment (bastelfreak) [reply data/Archlinux.yaml]: I think we can delete this. the mcollective package works only in combination with the puppet 5 agent.

File: data/FreeBSD.yaml (hunk 2 above)
Comment (smortex) [reply data/FreeBSD.yaml]: Regarding FreeBSD: `rubygem-choria-mcorpc-support` was committed to the ports a few weeks ago, I am running this without issue, and I plan to drop the various legacy mcollective ports in early 2021… so I would recommend ensuring `rubygem-choria-mcorpc-support` is installed.

Regarding OpenBSD, I am not a user ATM, but [I only see a mcollective package here](https://ftp.openbsd.org/pub/OpenBSD/6.8/packages/amd64/), so sticking to `mcollective` is probably the safest.

**Pull request merged**